pyanaconda/anaconda_log.py

```diff
@@ -130,6 +137,25 @@ class AnacondaFileHandler(_AnacondaLogFixer, logging.FileHandler):
 class AnacondaStreamHandler(_AnacondaLogFixer, logging.StreamHandler):
     pass
 
+class AanacondaPrefixFilter(logging.Filter):
+    """Add a log_prefix field, which is based on the name property,
+    but without the "anaconda." prefix.
+
+    Also if name is equal to "anaconda", generally meaning some sort of
+    general (or miss-directed) log message, set the log_prefix to "misc".
+    """
+
+    def filter(self, record):
+        record.log_prefix = ""
+        if record.name:
+            # messages going to the generic "anaconda" logger get the log prefix "misc"
+            if record.name == "anaconda":
+                record.log_prefix = "misc"
+            elif record.name.startswith("anaconda."):
```

Comment:
I have looked at the logs and I miss the `anaconda` prefix a little :-) But I can get used to it.